### PR TITLE
Python 3 compatibility

### PR DESCRIPTION
**Pull request by mbertrand**
Some changes were required to gdal_functions.py to become python 3 compatible.

https://travis-ci.org/OpenDataAnalytics/gaia

### CODE REVIEW

File: gaia/gdal_functions.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def gdal_resize(raster, dimensions, projection, transform):
     return resized_ds
 
 
-def gdal_clip(raster_input, raster_output, polygon_json, nodata=-32768):
+def gdal_clip(raster_input, raster_output, polygon_json, nodata=0):
```
Comment (aashish24) [inline gaia/gdal_functions.py]: @mbertrand why this changed?

Comment (mbertrand) [inline gaia/gdal_functions.py]: 0 is a safer default nodata value to use if none is provided (if the input datasource is of type UInt16, -32768 gets converted to positive 32768).

**Comment by aashish24** [comment]
LGTM :+1:

**Comment by aashish24** [comment]
Thanks @mbertrand

**Pull request merged**